test/tests/kernels/vector_fe/tests

```diff
@@ -8,6 +8,17 @@
     issues = "#10049"
     requirement = "The system shall be able to solve a curl-curl problem using NEDELEC_ONE elements."
   [../]
+  [vector_fe_errors]
+    type = CSVDiff
+    input = vector_kernel.i
+    csvdiff = vector_kernel_out.csv
+    cli_args = 'Postprocessors/active="L2Error HCurlSemiError HCurlError" '
+               'Outputs/exodus=false Outputs/csv=true'
+    design = 'ElementVectorL2Error.md ElementHCurlSemiError.md ElementHCurlError.md'
+    issues = '#26394'
+    requirement = 'The system shall be able to estimate the L2 and H(curl) errors for a curl-curl '
+                  'problem solved using NEDELEC_ONE elements.'
```

Comment:
```suggestion
                  'problem solved using Nedelec elements of the first kind.'
```
You were probably going off the above bad example, but we try to keep implementation details out of requirements. From a thoretical point of view, we like to imagine that the requirement came before the implementation

Reply:
Done, and also fixed the "bad" examples in the same file in a separate commit.